codecparsers/nalReader.h

```diff
@@ -28,7 +28,9 @@ class NalReader : public BitReader
     NalReader(const uint8_t *data, uint32_t size);
 
     /*parse Exp-Golomb coding*/
+    bool readUe(uint32_t& v);
```

Comment:
We should use this version for every codec parser.  Maybe we need remove uint32_t readUe() after all codec parser called this function.